go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/metalmatze/digitalocean_exporter
+
+go 1.13
+
+require (
+	github.com/digitalocean/godo v1.29.0
```

Comment:
There should be more dependencies. Like the Prometheus client and a few others.

Reply:
One aspect of Go Modules I don't understand is being prescriptive about module versions; I would have not included even `godo`.

I've added them at your suggestion (including `go.sum`) but "cloners" will now forever be tied to these versions unless they explicity revise them.

It seems useful to specify module versions for releases but not for intermediate commits.

Resolved: https://github.com/metalmatze/digitalocean_exporter/pull/6/commits/38e74026b8d7c9b1a1cf18faebc22223d3c152ca

Reply:
Before Go modules we had the same, we even committed the vendor/ directory, so that we even had the same source code of dependencies. This just recreates that with the go.sum file, I guess.